src/modules/manageComments.js

```diff
@@ -0,0 +1,71 @@
+let commentsData = [];
+
+const loadFunction = async (appId, ID) => {
+  await getScore(appId, ID);
+
+  const parentElement = document.querySelector('.myArticle-style');
+  const divComments = document.createElement('div');
+  parentElement.appendChild(divComments);
+  divComments.classList.add('divComments');
+
+  divComments.innerHTML = `
+      <h2 class="h2Comments">Comments (${commentsData.length})<h2>
+      <ul class="ulComments">
+      </ul>
+    `;
+  const ulComments = document.querySelector('.ulComments');
+
+  commentsData.forEach((element) => {
+    const liComment = document.createElement('li');
+    ulComments.appendChild(liComment);
+    liComment.classList.add('liComments');
+    liComment.innerHTML = `
+    <h3>${element.creation_date} ${element.username} ${element.comment}</h3>
```

Comment:
```
<h3 class="singleComment">
  ${element.creation_date}: Comment by ${element.username} -  ${element.comment}
</h3>
```

Please consider adding a bit more info so it's easier to say what is what in the comment maybe, plus some styling to add padding between comments, so they are not that close to each other.